examples/evaluation_and_profiling/email_phishing_analyzer/configs/config_optimizer.yml

```diff
@@ -0,0 +1,132 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+general:
+  use_uvloop: true
+  telemetry:
+    logging:
+      console:
+        _type: console
+        level: WARN
+      file:
+        _type: file
+        path: ./.tmp/email_phishing_analyzer.log
+        level: DEBUG
+
+functions:
+  email_phishing_analyzer:
+    _type: email_phishing_analyzer
+    optimizable_params:
+      - llm
+      - prompt
+  prompt_init:
+    _type: prompt_init
+    optimizer_llm: prompt_optimizer
+    system_objective: Agent that triages an email to see if it is a phishing attempt or not.
+  prompt_recombination:
+    _type: prompt_recombiner
+    optimizer_llm: prompt_optimizer
+    system_objective: Agent that triages an email to see if it is a phishing attempt or not.
+
+llms:
+  llama_3_405:
+    _type: nim
+    model_name: meta/llama-3.1-405b-instruct
+    temperature: 0.0
+    max_tokens: 1024
+    optimizable_params:
+      - temperature
+      - top_p
+      - max_tokens
+  llama_3_70:
```

Comment:
Or would it just be a RuntimeError if the user tried to optimize a parameter which was unsupported for the model to begin with?